react/tic-tac-toe/src/app/screens/Login/layout.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Field, Form, reduxForm } from 'redux-form';
+import PropTypes from 'prop-types';
+
+import { hasLength, isRequired, isValidEmail } from '../../../utils/Validations';
+
+import styles from './styles.module.scss';
+import LoginInput from './components/LoginInput';
+
+function Login({ handleSubmit }) {
+  return (
+    <div className={`${styles.column} ${styles.middle}`}>
+      <h1 className={styles.loginTitle}>Login</h1>
+      <Form className={`${styles.column} ${styles.middle}`} onSubmit={handleSubmit}>
+        <label className={styles.loginDetail}>Email</label>
+        <Field
+          name="email"
+          type="text"
+          label="Email"
+          component={LoginInput}
+          validate={[isValidEmail, isRequired]}
```

Comment:
Small improvement, move the validations array to constants to you don't create an array each time this renders